app/models/comment.rb

```diff
@@ -3,5 +3,8 @@ class Comment < ApplicationRecord
   belongs_to :post
 
   validates :text, presence: true
+
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- **[OPTIONAL]** I would recommend you add an active record callback for this method